src/platform/packages/shared/kbn-apm-synthtrace-client/src/lib/utils/hash.ts

```diff
@@ -19,6 +18,19 @@ export function hashKeysOf<T extends Fields>(source: T, keys: Array<keyof T>) {
   return hashed;
 }
 
+// this hashing function has same output as fnv-plus
+function fnv1a32(str: string): number {
```

Comment:
Does this function have test cases in the original repo? Might be useful to port some in order to enforce that the output of this function should never change.